go.mod

```diff
@@ -4,28 +4,48 @@ go 1.12
 
 require (
 	github.com/Qitmeer/crypto v0.0.0-20200516043559-dd457edff06c
+	github.com/aristanetworks/goarista v0.0.0-20200812190859-4cb0e71f3c0e
+	github.com/btcsuite/btcd v0.20.1-beta
```

Comment:
should not depend on `btcd`

Reply:
https://github.com/Qitmeer/qitmeer/pull/382/commits/247260ab15756863809d6622ac5c1147b3cf2f77